api/controllers/WebViewController.js

```diff
@@ -108,15 +109,46 @@ module.exports = {
       zigzag: `zigzag?${utmSourceVeritone}&${utmMedium}&utm_campaign=Zigzag`,
     };
 
-    let originalName = req.url.split('/')[1];
-    let redirectName = podcastDirectories[originalName];
-    return res.redirect(
-      301,
-      `${sails.config.globals.premiumShineBaseUrl}/promo/${redirectName}`
-    );
-  },
+    Promise.coroutine(function*() {
```

Comment:
fwiw you should be able to use async/await on aurora. Heroku's been deploying this project with Node 10.